tests/test_invoice_tools.py

```
@@ -4,7 +4,7 @@
 
 import json
 import pytest
-from unittest.mock import patch, MagicMock
+from unittest.mock import patch, MagicMock, AsyncMock
```

Ruff (0.12.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import

(F401)